script/main.py

```diff
@@ -19,10 +104,120 @@ def init_argparser():
   return args
 
 
+def create_connection(db_file):
+  conn = None
+  try:
+    conn = sqlite3.connect(db_file)
+  except sqlite3.Error as e:
+    print(e)
+  
+  return conn
+
+
+def create_table(conn, create_table_sql):
+  try:
+    cursor = conn.cursor()
+    cursor.execute(create_table_sql)
+  except sqlite3.Error as e:
+    print(e)  
+
+
+def insert_users_to_table(conn, users):
+  sql = '''INSERT INTO users VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)'''
+  cur = conn.cursor()
+  for columns in people:
+    cur.execute(sql, columns)  
+  conn.commit()
+
+
 def init_db():  
-  '''Initializing the database'''
+  '''Initializing the database'''  
+
+  database = "db/pythonsqliteusers.db"
+  
+  sql_create_users_table = ''' CREATE TABLE IF NOT EXISTS users (
+    gender text,
+    name_title text,
+    name_first text,
+    name_last text,
+    location_street_number integer,
+    location_street_name text,
+    location_city text,
+    location_state text,
+    location_country text,
+    location_postcode integer,
+    location_coordinates_latitude numeric,
+    location_coordinates_longitude numeric,
+    location_timezone_offset numeric,
+    location_timezone_description text,
+    email text,
+    login_uuid text,
+    login_username text,
+    login_password text,
+    login_salt text,
+    login_md5 text,
+    login_sha1 text,
+    login_sha256 text,
+    dob_date text,
+    dob_age text,
+    registered_date text,
+    registered_age text,
+    phone text,
+    cell text,
+    id_name text,
+    id_value text,
+    picture_large text,
+    picture_medium text,
+    picture_thumbnail text,
+    nat
+  ); '''
 
-  print('Nothing is implemented')
+  conn = create_connection(database)  
```

Comment:
do not create connection here. connection to the database should be passed as an argument.